htdocs/core/lib/project.lib.php

```diff
@@ -1064,6 +1064,239 @@ function projectLinesa(&$inc, $parent, &$lines, &$level, $var, $showproject, &$t
 }
 
 
+/**
```

Comment:
Something wrong here, we should not have this file modified in this PR

Reply:
I pull develop into my branch KanbanExpedition and resolve the conflict in project.lib.php.
It should work properly